gdk/wizard/WizardChecker.py

```diff
@@ -77,3 +83,70 @@ def is_valid_custom_build_command(self, input_value):
             return False
         except ValueError:
             return len(input_value) > 0
+
+    def check_build_options(self, input_value):
```

Comment:
rename these `check_`* methods to `is_valid_*`.

Reply:
got it, changing now